src/Squidex.Domain.Apps.Entities/Contents/IContentEntity.cs

```diff
@@ -24,6 +24,8 @@ public interface IContentEntity :
 
         Status Status { get; }
 
+        Status2 Status2 { get; }
```

Comment:
We don't need that. The new status struct should be 100% backwords compattible.

Reply:
I''ll change that now.